infra/gcp/infra.yaml

```diff
@@ -325,3 +325,4 @@ infra:
       k8s-staging-storage-migrator:
       k8s-staging-test-infra:
       k8s-staging-txtdirect:
+      k8s-staging-cloud-provider-ibm:
```

Comment:
@ambiknai Please, respect the alphabetic order of this list.

Reply:
I have moved it as per this comment.